src/yarp.c

```diff
@@ -1926,6 +1926,24 @@ parse_expression_prefix(yp_parser_t *parser) {
 
       return array;
     }
+    case YP_TOKEN_PARENTHESIS_LEFT: {
+      yp_token_t opening = parser->previous;
+      yp_node_t *parentheses;
+
+      if (parser->current.type != YP_TOKEN_PARENTHESIS_RIGHT && parser->current.type != YP_TOKEN_EOF) {
+        yp_node_t *statements = yp_node_statements_create(parser);
+        yp_node_t *statement = parse_expression(parser, BINDING_POWER_DEFINED, "Could not parse content in parenthesis.");
```

Comment:
I think we're really close here! The thing is, it's not like an array because it actually allows multiple statements, so it's the lowest binding power.

You can also have something like:

```ruby
(
  a
  b
)
```

and it'll work. So in this case we should use `parse_statements` instead of `parse_expression`.